libs/client/client.ts

```diff
@@ -184,6 +186,7 @@ class Client {
    * @param {*} token An instance of Token holding token data such as contract address
    * @param {NightfallZkpKeys} ownerZkpKeys Sender's set of Zero-knowledge proof keys
    * @param {RecipientNightfallData} recipientNightfallData An object with [valueWei], [recipientCompressedZkpPublicKey]
+   * @param {string} tokenId The tokenId of the token to be transfered
```

Comment:
:rofl: now you make me feel pedantic

```suggestion
   * @param {string} tokenId The tokenId of the token to be transferred
```